From: Jason Rhinelander <jason@imaginary.ca>
Date: Fri, 30 Oct 2015 13:54:26 -0400
Subject: README: proper debian packages; no more cpack

The README instructions don't work anymore: cpack is gone in favour or
native debian packaging (under the debian* branches).
---
 README.md | 54 ++++++++++++++++++++++++++++++++++--------------------
 1 file changed, 34 insertions(+), 20 deletions(-)

diff --git a/README.md b/README.md
index 8198218..3ad889d 100644
--- a/README.md
+++ b/README.md
@@ -4,29 +4,57 @@
 
 Eris is a C++ library designed for simulating “economies“ consisting of agents
 who follow programmable rules, with the intention of deriving complex
-macro-level behaviour that is an emergent property of agent interaction.
+macro-level behaviour that is an emergent property of agent interaction.  This
+project provides no implementation itself, but is intended to be used as the
+basis of an agent-based model.  See the
+[creativity](https://git.imaginary.ca/eris/creativity/) project for a working
+example of a moderately complex ABM built using eris.
 
 This project was motivated by the inadequacy of mathematical models, which
 often impose severe constraints on agents in the name of tractability.
 
 At its core, this library imposes few constraints on the behaviour of agents.
 Individuals agents can, for instance, be programmed to be ultra-rational, or
-can be “dumb” in the sense of following only simple rules of thumb.
+can be “dumb” in the sense of following only simple rules of thumb.  The
+behaviours and means of interacting are left to packages developed using this
+library.
 
 Beyond the core, the library offers many specialized implementations that may
-be used as appropriate, but can equally well be replaced with alternative
-implementations.
+be used as appropriate, but can equally well be ignored, enhanced, or replaced
+with alternative implementations.
 
 The library name, Eris, is the name of the Greek goddess of chaos.
 
 ## Requirements
 
-- [boost](http://www.boost.org/); only the Math component is needed (and only
-  during compilation).
+- [boost](http://www.boost.org/); only the Math component is needed.
 - [Eigen](http://eigen.tuxfamily.org/)
 - A C++ compiler supporting the C++11 standard, such as
   [clang](http://clang.llvm.org/) (3.3+) or [g++](https://gcc.gnu.org/) (4.9+)
 
+## Installing (Debian-based systems)
+
+Regularly updated library, header, and documentation packages are available for
+amd64 systems by adding the following repository to /etc/apt/sources.list:
+
+    deb https://imaginary.ca/debian sid main
+
+(for Debian testing or unstable), or
+
+    deb https://imaginary.ca/debian jessie main
+
+(for the current Debian stable release).
+
+The required key signature for package verification can be installed using:
+
+    curl -s https://imaginary.ca/public.gpg | sudo apt-key add -
+
+You generally want to install the liberis-dev package and possibly the eris-doc
+package.
+
+These packages are build from the "debian" and "debian-jessie" branch of the
+package repository.
+
 ## Compiling
 
 To compile on a unix-like system, create a new build directory somewhere, then
@@ -39,20 +67,6 @@ You can install directly to the system (usually under /usr/local) using:
 
     make install
 
-or alternatively build a .deb package to install using one of:
-
-    make package
-
-followed by an appropriate package command to install the resulting package
-(for example, on Debian/Ubuntu: dpkg -i eris_x.y.z~gityyyymmdd~tag_amd64.deb).
-
-You may also be able to use:
-
-    cpack -G RPM
-
-to generate a .rpm package instead of a .deb, but this functionality is
-untested by the author.
-
 ## License
 
 This program is free software: you can redistribute it and/or modify
